excercise-e-shop/assets/scss/base/_mixin.scss

```diff
@@ -0,0 +1,11 @@
+@mixin badge-sale($left) {
```

Comment:
`mixin` need default value for arguments.
example: `@mixin badge-sale($left: left)`